src/averagePerceptronTagger.jl

```diff
@@ -0,0 +1,326 @@
+using DataStructures
+using Random
+using BSON
+
+"""AVERAGE PERCEPTRON MODEL"""
+
+"""
+This module contains the Average Perceptron model
+which was original implemented by Matthew Honnibal.
+
+The model learns by basic perceptron algorithm
+but after all iterations weights are being averaged
+"""
+mutable struct AveragePerceptron
+    classes :: Set
+    weights :: Dict
+    _totals :: DefaultDict
+    _tstamps :: DefaultDict
+    i :: Int64
+    START :: Array

```

Comment:
I have named it in capitals because it is a constant which contains `["-START-", "-START2-"]`. And it is not changed through out the program. Same for the `END` which contains `["-END-", "-END2-"]`.